src/watchdog.rs

```diff
@@ -0,0 +1,98 @@
+use crate::hal::watchdog::{Watchdog, WatchdogEnable};
+
+use crate::stm32::{DBGMCU, IWDG};
```

Comment:
```suggestion
use crate::stm32::{DBGMCU, IWDG};
use crate::time::MilliSeconds;
```

And add a milliseconds implementation to `src/time.rs` :wink: